lib/health_cards/verifiable_credential.rb

```diff
@@ -44,6 +47,14 @@ def credential
 
     def jwt; end
 
+    def save_as_qrcode
+      qrcode = RQRCode::QRCode.new(credential.to_json, size: 40)
```

Comment:
I think we need to set `mode: number` too to make it a numeric QR Code.

[Every Health Card can be embedded in a QR Code](https://smarthealth.cards/#numerical-encoding) section of the spec has some details on the QR code settings that we should double check.

Reply:
Thanks, I did not see this QR code section.

Reply:
It's a pretty new section so I don't think it was actually around when we originally created this task.